exercises/practice/goldbach/.docs/instructions.md

```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+[Goldbach's Conjecture](https://en.wikipedia.org/wiki/Goldbach%27s_conjecture) is
+an old mathematical conjecture (a statement that has held true for demonstrated
+cases and has never been proved false, but has never been proved true) that states
+that any even number greater than two can be expressed as the sum of two prime numbers.
+Consider 4 (2 + 2), 6 (3 + 3), 8 (3 + 5), and 10 (5 + 5, 3 + 7). Write a z3 function to
```

Comment:
I don't think that you need to say "Write a Z3 function ..." since this will be under the Z3 track. Maybe just start the sentence with "Write a function ..."